apps/healthlocker/web/controllers/oxleas_adhd/about_me_controller.ex

```diff
@@ -1,13 +1,94 @@
 defmodule Healthlocker.OxleasAdhd.AboutMeController do
+  alias Healthlocker.{AboutMe, User}
   use Healthlocker.Web, :controller
 
-  def new(conn, params) do
-    conn
-    |> render("new.html")
+  def new(conn, %{"user_id" => user_id}) do
+    user = Repo.get!(User, user_id)
+    changeset = AboutMe.changeset(%AboutMe{})
+    about_me_query = from am in AboutMe, where: am.user_id == ^user_id
```

Comment:
I think we could also use the ```Repo.get_by/2``` function: https://hexdocs.pm/ecto/Ecto.Repo.html#c:get_by/3